PluginsMakefile.mk

```diff
@@ -103,7 +115,14 @@ npm: ## Run a npm command, example: make npm c='install mypackage/package'
 .PHONY: npm
 
 ##—— Testing and static analysis ———————————————————————————————————————————————
-verify: license-headers-check parallel-lint phpstan phpcsfixer-check phpunit ## Run all our lints/tests/static analysis
+verify:  ## Run all our lints/tests/static analysis
+	@$(call run_if_exists, tools/HEADER, license-headers-check)
+	@$(call run_always, parallel-lint)
+	@$(call run_if_exists, phpstan.neon, phpstan)
+	@$(call run_if_exists, .php-cs-fixer.php, phpcsfixer-check)
+	@$(call run_if_exists, rector.php, rector-check)
+	@$(call run_if_exists, psalm.xml, psalm)
+	@$(call run_if_exists, phpunit.xml, phpunit)
```

Comment:
We could reorder these a bit I think, for example psalm is kinda slow and is unlikely to fail as it cover only taint so it should be the last step IMO.

I think the first steps should be the ones that are most likely to fail, which is phpcsfixer, phpunit and then phpstan in my opinion (maybe parallel-lint need to be kept before them tho).